.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: java
+
+jdk:
+  - oraclejdk11
+
+cache:
+  directories:
+    - .autoconf
+    - $HOME/.m2
+
+install: true
+script:
+  - cd jaxb-ri
+  - mvn -U -C -Pstaging clean install
```

Comment:
I think in script it's better to use long options (i.e. `--strict-checksums` instead of short `-C`, etc.), as they are more readable.